config.yml

```diff
@@ -22,13 +22,17 @@ client:
 
 faucet:
   name: bob
-  coins: ["1000000mand", "10000000stake"]
+  coins: ["1000000mand", "10000000cred"]
 
 genesis:
   app_state:
     staking:
       params:
+        bond_denom: "cred"
         unbonding_time: "0.001s"
+    mint:
+      params:
+        mint_denom: "cred"
     bank:
       params:
-        send_enabled: [{"denom": "stake","enabled": false}]
+        send_enabled: [{"denom": "cred","enabled": false}]
```

Comment:
In genesis file, `stake` is being used in `crisis` and `gov` module as well. I guess we need to update them to `mand`

Reply:
Done